mypyc/irbuild/specialize.py

```diff
@@ -36,20 +37,21 @@
 #
 # Specializers take three arguments: the IRBuilder, the CallExpr being
 # compiled, and the RefExpr that is the left hand side of the call.
+
 Specializer = Callable[['IRBuilder', CallExpr, RefExpr], Optional[Value]]
 
 # Dictionary containing all configured specializers.
 #
 # Specializers can operate on methods as well, and are keyed on the
 # name and RType in that case.
-specializers = {}  # type: Dict[Tuple[str, Optional[RType]], Specializer]
+specializers = defaultdict(list)  # type: DefaultDict[Tuple[str, Optional[RType]], List[Specializer]]
```

Comment:
I think that this change is not needed.

Reply:
I think the `defaultdict` is not needed while making the candidates specializers a list is necessary, otherwise in this case, `translate_safe_generator_call` and `faster_min` would compete with each other.